src/components/typography/caption/index.tsx

```diff
@@ -1,25 +1,31 @@
 import qtMerge from 'qtMerge'
-import { forwardRef } from 'react'
-import { BodyTypographyProps } from 'types'
+import { HTMLAttributes, forwardRef } from 'react'
+import { BodyTextProps } from '../text/text.classes'
 
-export const CaptionText = forwardRef<
-  HTMLParagraphElement,
-  Omit<BodyTypographyProps, 'size'>
->(({ bold, italic, underline, className, ...rest }, ref) => {
-  return (
-    <p
-      ref={ref}
-      className={qtMerge(
-        'text-caption text-typography-prominent',
-        bold ? 'font-bold' : 'font-regular',
-        italic ? 'italic' : 'not-italic',
-        underline && 'underline',
-        className,
-      )}
-      {...rest}
-    />
-  )
-})
+export interface CaptionTextProps
+  extends HTMLAttributes<HTMLParagraphElement>,
+    BodyTextProps {}
+
+export const captionTextSizeClassnames = 'text-caption'
+
+export const CaptionText = forwardRef<HTMLParagraphElement, CaptionTextProps>(
+  ({ bold, italic, underline, className, ...rest }, ref) => {
+    return (
+      <p
+        ref={ref}
+        className={qtMerge(
+          captionTextSizeClassnames,
+          'text-typography-prominent',
+          bold ? 'font-bold' : 'font-regular',
+          italic ? 'italic' : 'not-italic',
+          underline && 'underline',
```

Comment:
Do we need these? We can just pass className here.

Reply:
yeah man, we need to have them as props to be close to design.